flutter-for-xamarin-forms.md

```diff
@@ -0,0 +1,2322 @@
+   GlobalMaterialLocalizations.delegate,
+   GlobalWidgetsLocalizations.delegate,
+ ],
+ supportedLocales: [
+    const Locale('en', 'US'), // English
+    const Locale('he', 'IL'), // Hebrew
+    // ... other locales the app supports
+  ],
+  // ...
+)
+{% endprettify %}
+
+The delegates contain the actual localized values, while the `supportedLocales`
+defines which locales the app supports. The above example uses a `MaterialApp`,
+so it has both a `GlobalWidgetsLocalizations` for the base
+widgets localized values, and a `MaterialWidgetsLocalizations` for the Material
+widgets localizations. If you use `WidgetsApp` for your app, you don't
+need the latter. Note that these two delegates contain "default"
+values, but you'll need to provide one or more delegates for your own app's
+localizable copy, if you want those to be localized too.
+
+When initialized, the `WidgetsApp` (or `MaterialApp`) creates a
+[`Localizations`](https://docs.flutter.io/flutter/widgets/Localizations-class.html)
+widget for you, with the delegates you specify.
+The current locale for the device is always accessible from the `Localizations`
+widget from the current context (in the form of a `Locale` object), or using the
+[`Window.locale`](https://docs.flutter.io/flutter/dart-ui/Window/locale.html).
+
+To access localized resources, use the `Localizations.of()` method to
+access a specific localizations class that is provided by a given delegate.
+Use the [`intl_translation`](https://pub.dartlang.org/packages/intl_translation)
+package to extract translatable copy to
+[arb](https://code.google.com/p/arb/wiki/ApplicationResourceBundleSpecification)
+files for translating, and importing them back into the app for using them
+with `intl`.
+
+For further details on internationalization and localization in Flutter, see the
+[internationalization guide](/tutorials/internationalization),
+which has sample code with and without the `intl` package.
+
+## Where is my project file?
+
+In Xamarin.Forms you will have a `csproj` file. Flutter doesn't have a project file.
+Most similar to this, is pubspec.yaml, which contains package dependencies and 
+various project details.
+
+## What is the equivalent of Nuget? How do I add dependencies?
+
+In the .NET eco-system, native Xamarin projects and Xamarin.Forms projects had access
+to Nuget and the inbuilt package management system. Flutter apps contain a native 
+Android app, native iOS app and Flutter app.
+
+In Android, you add dependencies by adding to your Gradle build script. In iOS, you add 
+dependencies by adding to your `Podfile`.
+
+Flutter uses Dart's own build system, and the Pub package manager.
+The tools delegate the building of the native Android and iOS wrapper apps to the
+respective build systems.
+
+In general, use `pubspec.yaml` to declare external dependencies to use in Flutter. A good 
+place to find Flutter packages is [Pub](https://pub.dartlang.org/flutter/packages/).

```

Comment:
Ha! Another example where I added this above.

I would use pub.dartlang.org/flutter as the link.